frontend/src/scenes/users/PeopleTable.js

```diff
@@ -1,107 +1,75 @@
-import React, { useState } from 'react'
-import { Loading } from 'lib/utils'
+import React, { useEffect, useState } from 'react'
+import { Button, Table } from 'antd'
+import api from 'lib/api'
 import { Link } from 'react-router-dom'
-import moment from 'moment'
+import { FilterLink } from '../../lib/components/FilterLink'
+import { DeleteOutlined } from '@ant-design/icons'
 
-function FilterLink({ property, value, onClick }) {
-    const label = typeof value === 'object' ? JSON.stringify(value) : value
+import { toast } from 'react-toastify'
 
-    if (onClick) {
-        return (
-            <a
-                href="#"
-                onClick={e => {
-                    e.preventDefault()
-                    onClick(property, value)
-                }}
-            >
-                {label}
-            </a>
-        )
-    } else {
-        return <span>{label}</span>
-    }
+function Properties({ properties }) {
+    return (
+        <div className="d-flex flex-wrap flex-column" style={{ maxHeight: 200 }}>
+            {Object.keys(properties)
+                .sort()
+                .map(key => (
+                    <div
+                        style={{
+                            flex: '0 1',
+                        }}
+                        key={key}
+                    >
+                        <strong>{key}:</strong> {properties[key]}
+                    </div>
+                ))}
+        </div>
+    )
 }
 
-export function PeopleTable({ loading, people, onClickProperty }) {
-    const [personSelected, setPersonSelected] = useState(null)
-
+export function PeopleTable({ people, loading, actions, onChange }) {
+    let columns = [
+        {
+            title: 'Person',
+            dataIndex: 'name',
+            key: 'name',
+            render: (_, person) => (
+                <Link to={'/person/' + encodeURIComponent(person.distinct_ids[0])} className="ph-no-capture">
+                    {person.name}
+                </Link>
+            ),
+        },
+        actions && {
+            title: 'Actions',
+            render: person => (
+                <Button
+                    className="float-right"
```

Comment:
if you remove this line, the 'actions' label at the top of the column will be aligned with the action icon